paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetState.kt

```diff
@@ -16,15 +16,13 @@ internal sealed interface PaymentSheetState : Parcelable {
     data class Full(
         val config: PaymentSheet.Configuration,
         val customer: CustomerState?,
-        val isGooglePayReady: Boolean,
         val linkState: LinkState?,
-        val isEligibleForCardBrandChoice: Boolean,
```

Comment:
This was already being tracked in PaymentMethodMetadata, so I removed it in favor of that one.